.github/workflows/build-container.yaml

```
@@ -66,6 +66,18 @@ jobs:
             type=semver,pattern=v{{major}}.{{minor}}
             type=semver,pattern=v{{major}}.{{minor}}.{{patch}}
 
+      - name: Switch to matching config branch
+        run: |
+          cd configs
+          git checkout ${{ github.ref_name }} || true
+
+      - name: Generate rootfs
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:3:30: Quote this to prevent word splitting

(shellcheck)